lib/departure/dsn.rb

```diff
@@ -9,16 +9,17 @@ class DSN
     def initialize(database, table_name)
       @database = database
       @table_name = table_name
+      @added_dsn_items = ENV.fetch('PERCONA_ADDED_DSN_ITEMS', nil)
```

Comment:
What do you think about this alternative phrasing?

```suggestion
      @suffix = ENV.fetch('PERCONA_DSN_SUFFIX', nil)
```

Reply:
I knew that variable was clunky and am happy to update it to something less clunky.  I'll update this when I add a test and CHANGELOG entry (per your other comment).